.circleci/config.yml

```diff
@@ -662,9 +662,10 @@ jobs:
 
       # Ensure everything got cleaned up successfully
       - run:
-          name: "Verify only the engine container remains after the clean"
+          # Once persistent volume is implemented, log aggregator will be cleaned as well, for now it remains
+          name: "Verify only the logs aggregator remains after the clean"
```

Comment:
Why do we need to keep the log aggregator after the clean?

Reply:
we don't, ill clean it